### PR TITLE
Email Show on User Profile Page

### PR DESCRIPTION
**Pull request by seveibar**
This project has been [raided](https://www.twitch.tv/opensourceraiders)! Works locally.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,4 +18,6 @@ services:
       DATABASE_URL: "ecto://postgres:postgres@db/postgres"
   db:
     image: postgres:alpine
-    container_name: papercups
```
Comment (agundy) [inline docker-compose.yml:21]: I don't know what this was doing but I couldn't get the docker-compose to work, complained about not having a user setup.

Comment (reichert621) [reply docker-compose.yml:21]: cc @cheeseblubber do you know anything about this?

**Comment by reichert621** [comment]
thanks for tackling this @seveibar! mind providing a screenshot of your update?

**Comment by agundy** [comment]
Here's a screenshot. I didn't get a chance to get the db fully set up so I could dev but couldn't run tests, I think the tests are not going to work since I changed the api though.
view

**Comment by reichert621** [comment]
hey @agundy -- is the main issue that the docker stuff isn't working correctly at the moment?

**Comment by reichert621** [comment]
closing, main fix handled in https://github.com/papercups-io/papercups/pull/219

feel free to submit a PR for fixes to the docker script though! (cc @cheeseblubber fyi)

**Pull request closed without merging**